src/main/java/org/broadinstitute/consent/http/mail/message/DarExpiredMessage.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.consent.http.mail.message;
+
+import java.util.Map;
+import org.broadinstitute.consent.http.enumeration.EmailType;
+import org.broadinstitute.consent.http.models.User;
+
+public class DarExpiredMessage extends MailMessage {
+
+  private final String darCode;
+
+  public DarExpiredMessage(User toUser, String darCode) {
+    super(toUser, EmailType.DAC_EXPIRED);
+    this.darCode = darCode;
+  }
+
+  @Override
+  public String createSubject() {
+    return "Data Access Request Expired in DUOS";
```

Comment:
The ticket didn't specify the subject line so I looked at other messages to try and copy the existing usage.